checkoff

```diff
@@ -1,167 +1,162 @@
 #!/usr/bin/env python3
 import argparse
+import getpass
 import json
-import os
 import subprocess
 import sys
 
 from ocflib.account.search import user_exists
+from ocflib.infra.db import get_connection
 
+MYSQL_PASSWORD_FILE = 'mysql_pwd'
+DATA_PATH = 'checkoff_data.json'
```

Comment:
confused why we need this if we're storing things in mysql. shouldn't we be storing student data etc. in mysql as well?

Reply:
because it's static data that never changes, it just makes for more and uglier queries to put this data in mysql, even if we denormalize the tables. I thought about this for a while and then decided it was simpler to just keep this sort of stuff local.

Reply:
agreed that the queries would get uglier. but it feels kinda strange having 2 different data sources like this, esp if we lose the checkoff file or someone edits it wrong for some reason.

im not too opposed to this right now but in the future if we want to use mysql then we should just change this to keep everything in mysql

Reply:
I've replaced it with mysql as the store for all the data